lte/protos/oai/nas_state.proto

```diff
@@ -475,4 +479,68 @@ message EmmContext {
   NewAttachInfo new_attach_info = 67;
   UeAdditionalSecurityCapability ue_additional_security_capability = 68;
   // TODO: add remaining emm_context elements
+
+  // 5G Guti
+  Guti_m5 m5_guti = 91;
+  Guti_m5 m5_old_guti = 92;
+  // 5G proto
+  map<uint32,SmfContext> smf_context_m =93;
```

Comment:
What is the key for this hashmap? Please use a field name which includes what the key is.

Reply:
+1

Reply:
Addressed this review comment.